packages/react-core/src/components/OptionsMenu/OptionsMenuToggleWithText.tsx

```diff
@@ -70,7 +70,7 @@ export const OptionsMenuToggleWithText: React.FunctionComponent<OptionsMenuToggl
       aria-expanded={isOpen}
       onClick={() => onToggle(!isOpen)}
     >
-      {toggleButtonContents}
+      <span className="pf-c-options-menu__toggle-button-icon">{toggleButtonContents}</span>
```

Comment:
Why is the `pf-c-options-menu__toggle-button-icon` hardcoded?

Reply:
Replaced hard-coded className with react-styles variable.